crates/teloxide-core/src/types/story.rs

```diff
@@ -1,5 +1,81 @@
 use serde::{Deserialize, Serialize};
 
-/// TBA 6.8: currently it holds no information
+use crate::types::{Chat, ChatKind, StoryId};
+
+/// This object represents a story.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub struct Story {}
+pub struct Story {
+    /// Chat that posted the story.
+    pub chat: Chat,
+
+    /// Unique identifier for the story in the chat.
+    pub id: StoryId,
+}
+
+impl Story {
+    /// Returns an URL that links to the story with it's id and chat username in
+    /// the form of `tg://resolve?domain=<…>&story=<…>`.
```

Comment:
I tried to search for this format, but found nothing.

I tried to copy link to some story (the one at the top of the ui, not in a chat) and got `https://t.me/{domain}/s/{id}`. 

Can both be correct?

Reply:
Yes, they are both correct. I checked this again, `tg://resolve?domain={domain}&story={id}` and `https://t.me/{domain}/s/{id}` work both.